### PR TITLE
add 6.4.5 changelog

### PR DESCRIPTION
**Pull request by hzoo**
wip since we'll probably merge more

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,56 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
...
+
+* **Documentation**
+ * `babel-traverse`: [#3269](https://github.com/babel/babel/pull/3269) Document visitors.explode. Thanks @forivall!
+
+* **Internal**
+ * `babel-plugin-transform-es2015-parameters`: [#3263](https://github.com/babel/babel/pull/3263) Test coverage.
+ * `babel-core`: [#3268](https://github.com/babel/babel/pull/3268) Add a test for ([T2892](https://phabricator.babeljs.io/T2892)).
+ * [#3275](https://github.com/babel/babel/pull/3275) Temporarily change flow types to fix lint.
+ * [#3277](https://github.com/babel/babel/pull/3277) Use local bin references instead of implied global in Makefile.
+ * [#3278](https://github.com/babel/babel/pull/3278) Fixup Makefile `.bin` references. Thanks @charliesome!
```
Comment (jmm) [inline CHANGELOG.md]: @hzoo *charliesome*'s PR is 3277.

Comment (hzoo) [inline CHANGELOG.md]: :cry: yeah I figured I would get some numbers off/wrong.

Comment (jmm) [inline CHANGELOG.md]: A small error :smiley: You've been doing an awesome job with these!

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,62 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
...
+ * `babel-helper-remap-async-to-generator`: [#3257](https://github.com/babel/babel/pull/3257) Fix issue with using `this` inside an arrow function ([T2765](https://phabricator.babeljs.io/T2765)). Thanks @horpto!
+   - This fixes:
+    ```js
+    class A {
+      async method() {
+        () => this; // this `this` wasn't being transpiled correctly
+      }
+    }
+    ```
+ * `babylon`: [#3272](https://github.com/babel/babel/pull/3272) Dedupe parser opts from passsed in multiple times. ([T3084](https://phabricator.babeljs.io/T3084)). Thanks @AgentME!
```
Comment (forivall) [inline CHANGELOG.md]: typo: "passsed"

**Pull request merged**